contracts/utils/deploy.js

```diff
@@ -159,14 +159,14 @@ const executeProposalOnFork = async (proposalId) => {
 
   const governor = await ethers.getContract("Governor");
 
-  // First enqueue the proposal, then execute it.
-  await withConfirmation(governor.connect(sGuardian).queue(proposalId, await getTxOpts()));
+  //First enqueue the proposal, then execute it.
+  await withConfirmation(governor.connect(sGuardian).queue(proposalId, await getTxOpts(executeGasLimit)));
```

Comment:
nit: does the executeGasLimit need to be applied to the queue as well?

Reply:
Good point, removing